lib/cmd/fh3/auth.js

```diff
@@ -0,0 +1,43 @@
+/* globals i18n */
+var fhreq = require("../../utils/request");
+var common = require("../../common.js");
+
+module.exports = {
+  'desc' : i18n._('Performs an auth call against an Auth Policy'),
+  'examples' : [{
+    cmd : 'fhc auth --policy=<policy> --app=<app> --device=<device> --params=<params>',
+    desc : i18n._('Performs a auth call against  Auth Policy')}],
+  'demand' : ['policy', 'app', 'device', 'params'],
```

Comment:
are `params` required? Should be?

Reply:
This is the way that it was before. I think so. What do you think? Is it making sense?